app/views/tags/edit.html.erb

```diff
@@ -12,6 +12,12 @@
                  },
                  class: "app-c-contextual-guidance__form" do %>
       <% @edition.document_type.tags.each do |tag_field| %>
+        <% if @edition.access_limit.present? && tag_field.id == "primary_publishing_organisation" %>
```

Comment:
It's a bit strange that this only activates for the primary org field, knowing that the supporting org field also affects this behaviour. On the other hand, I get that we don't necessarily want to show the same warning twice. This makes me wonder if we should actually have it as part of the intro text.